shared-libs/transitions/src/lib/schedules.js

```diff
@@ -93,7 +93,13 @@ module.exports = {
       return false;
     }
 
-    const docStart = objectPath.get(doc, schedule.start_from);
+    // start_from default to 'reported_date' it isn't set or 'undefined'
```

Comment:
Wondering if this comment is necessary, as the code is quite self-explanatory. 
Per [CHT guidelines](https://docs.communityhealthtoolkit.org/contribute/code/workflow/):

> comments are useful when they explain why some code exists, and should not be explaining what some code is doing.

Reply:
Okay, I've removed it.